<launch>

<!--all motor node-->

<node name = "talker" pkg = "liancheng_socket" type = "talk_test" output = "screen">
</node>

<node name = "serial" pkg = "liancheng_socket" type = "liancheng_serial" output = "screen">
</node>

<node name = "servo" pkg = "servo_wrist" type = "wrist_ctrl" args = "/dev/ttyUSB1" output = "screen">
</node>

<node name = "relay" pkg = "relay" type = "listener" output = "screen">
</node>

</launch>
